src/BartIO.jl

```diff
@@ -1,19 +1,45 @@
 module BartIO
 
 using BufferedStreams
-using PyCall
 
 # Exported functions
-export read_cfl
-export write_cfl
-export wrapper_bart
+export read_cfl, write_cfl, read_recon_header
+export set_bart_path, get_bart_path, bart
+
+
+"""
+    set_bart_path(pathToBart::String)
+
+Define the path to the BART toolbox (store in ENV["TOOLBOX_PATH"])
+"""
+function set_bart_path(pathToBart::String)
+    ENV["TOOLBOX_PATH"]=pathToBart
+    return pathToBart
+end
+
+function get_bart_path()
+        # Check bart toolbox path
+        bart_path = ENV["TOOLBOX_PATH"];
```

Comment:
You don't need semi-colons at the end of the line and, I think, it is discouraged in Julia.